CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 * Wrong assertion for user setting's `width` property when creating a new detail in separate route.
 * Rolling back isDeleted state of model if errors occurrs during destroying of the model.
 
+### Removed
+* Removed outdated style in `object-list-view` styles component.
```

Comment:
In previouse releases some changes were described in imperative present tense, some in past tense.
We don't have strict rule for changes descriptions, so it is insignificantly for a while.
I think we need to discuss it.